integration/app/test-demo/src/components/TestApps.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const TestApps: React.FC = ({ children }) => <ul>{children}</ul>;
+
+export default TestApps;
```

Comment:
We probably dont need a component for this - we can just use <ul/>

Reply:
Yeah I think it's good to keep it. 
It's weird to me that we have leave one native HTML element here.

```tsx
<Heading level={4}>React Component Library Test Demo</Heading>
  <ul>
  <TestApp
    name="Roster Test"
    route={routes.ROSTER_TEST}
    components={["RosterAttendee"]}
    hooks={[
      "useRosterState",
      "useAttendeeStatus",
      "useAttendeeAudioStatus",
      "useAudioVideo",
    ]}
  />
</ul>
);
```

Reply:
TestApps is more straightforward. When other people create new Test Apps, they know where to add them at once.

It's like:
```tsx
<Routes>
  <Route/>
  <Route/>
  ...
<Routes/>
```

Reply:
We should call it TestAppsList then - but I don't like having an entire component just for a <ul> tag, doesnt really bring any value

Reply:
Agree. Deleted `TestApps` component and used `<ul>` instead.